### PR TITLE
Floyd's Algorithm added

### PR DESCRIPTION
**Pull request by muddhit**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #226 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

File: Java/NthDigitMaxPrime/MaxPrime.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-imjort java.util.*; 
+import java.util.*;
```
Comment (yashaswibiyahut) [inline Java/NthDigitMaxPrime/MaxPrime.java]: Remove this file maxprime.java

File: Java/FloydAlgorithm/FloydAlgo.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        next = null;
+    }
+}
+public class DetectLoop
+{
+    public static void createLoop(Node head, Node tail, int k){
+        if (x == 0) return;
+        Node curr = head;
+
+        for(int i=1; i<k; i++)
```
Comment (yashaswibiyahut) [inline Java/FloydAlgorithm/FloydAlgo.java:23]: Add proper comment in your code.

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]

File: Java/FloydAlgorithm/FloydAlgo.java (hunk 2 above)
Comment (muddhit) [reply Java/FloydAlgorithm/FloydAlgo.java:23]: added.

**Comment by muddhit** [comment]
did the required changes @yashaswibiyahut

**Comment by yashaswibiyahut** [comment]
@Mukulbaid63 file still exists remove the file you only delete the coding inside it not deleted the file.

**Comment by muddhit** [comment]
@yashaswibiyahut i dont why but the delete option is disabled so let me create a new PR.

**Pull request closed without merging**